lemur/plugins/lemur_aws/plugin.py

```diff
@@ -750,3 +750,85 @@ def send(self, notification_type, message, excluded_targets, options, **kwargs):
 
         current_app.logger.info(f"Publishing {notification_type} notification to topic {topic_arn}")
         sns.publish(topic_arn, message, notification_type, options, region_name=self.get_option("region", options))
+
+
+class AWSACMSourcePlugin(SourcePlugin):
+    title = "AWS-ACM"
+    slug = "aws-acm-source"
+    description = "Discovers all ACM SSL certificates in an AWS account"
+    version = aws.VERSION
+
+    author_url = "https://github.com/netflix/lemur"
+
+    options = [
+        {
+            "name": "accountNumber",
+            "type": "str",
+            "required": True,
+            "validation": check_validation("^[0-9]{12,12}$"),
+            "helpMessage": "Must be a valid AWS account number!",
+        },
+        {
+            "name": "regions",
+            "type": "str",
+            "helpMessage": "Comma separated list of regions to search in, if no region is specified we look in all regions.",
+        },
+    ]
+
+    def get_certificates(self, options, **kwargs):
+        cert_data = acm.get_all_certificates(
+            account_number=self.get_option("accountNumber", options)
+        )
+
+        return [
+            dict(
+                body=c["Certificate"],
+                chain=c.get("CertificateChain"),
+                name=c["name"],
+                external_id=c["id"],
+            )
+            for c in cert_data
+        ]
+
+
+class ACMDestinationPlugin(DestinationPlugin):
```

Comment:
Can we add a couple of unit tests? e.g. destination upload resulting in source returning the cert, and destination clean resulting in source plugin not returning the cert

Reply:
@jmcrawford45 Could you explain more in details this?

> and destination clean resulting in source plugin not returning the cert

Reply:
Done!